main.py

```diff
@@ -92,18 +98,19 @@ def select_song(playlist_dict):
     return selected
 
 
-def play_song(songname):
-    songname = "/opt/mindfulness/song.mkv"
-    return play_mp3(songname, 60 * 10)
+def play_song():
+    return play_mp3(SONG_PLAY_PATH, 60 * 10)
 
 
 def download_song(songname):
-    local_song = "/opt/mindfulness/song"
+    local_song = "%s/song" % BASE_PATH
     os.system("youtube-dl %s -o %s" % (songname, local_song))
-    if os.path.exists("/opt/mindfulness/song"):
-       os.rename("/opt/mindfulness/song", "/opt/mindfulness/song.mkv")
-    logging.info("Downloaded %s to /opt/mindfulness/song.mkv" % songname)
-    return os.path.exists("/opt/mindfulness/song.mkv")
+    if os.path.exists(SONG_PLAY_PATH):
+        os.remove(SONG_PLAY_PATH)
```

Comment:
We DO want to remove `song.mkv` (SONG_PLAY_PATH), but of course not the file just downloaded called `song`.

This is because we are about to copy `song` to `song.mkv`, and this raises an OSError on Windows if the file already exists!